scripts/filter.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Isn't this file always generated from `filter.json.in` ? hence unnecessary to check it in...